lib/ansible/module_utils/network/ios/utils/utils.py

```diff
@@ -138,6 +138,8 @@ def validate_ipv6(value, module):
 def validate_n_expand_ipv4(module, want):
     # Check if input IPV4 is valid IP and expand IPV4 with its subnet mask
     ip_addr_want = want.get('address')
+    if len(ip_addr_want.split(' ')) > 1:
+        return ip_addr_want
```

Comment:
I think I can merge this as it does fix the error being generated.... it seems like this opens up a different problem, though.  This function documents that it will take an IPv4 address and subnet mask, validate it, and expand it. This is skipping the validation step so things like this could now be accepted, causing issues in other code: `want = "127.0.0.1/21 1.0.0.0"`  or even `want = "127 elephants party!"`

It seems like this branch of the code should split on space, then validate that:

* There are only two fields
* The first field is a valid ipv4 address
* The second field is a valid netmask
* Then return ip_addr_want as you're doing now.

Reply:
Talked with @justjais on slack.  He'll make these changes in a separate PR and backport that as well.  I'll merge this for now to fix the error.